app/web_ui/src/routes/(app)/settings/manage_tools/[project_id]/empty_tools.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
```

Comment:
Oh there's a new component for this `<Intro>` in intro.ts. Use that. Not all older ones are migrated to it, but we should use it going forward.